src/extension.ts

```diff
@@ -91,7 +89,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
     registerErrorHandler((info, error) => {
         const defaultMessage = localize('AWS.generic.message.error', 'Failed to run command: {0}', info.id)
-        handleError(error, info.id, defaultMessage)
+        logAndShowError(error, info.id, defaultMessage)
     })
```

Comment:
if we use the same `registerErrorHandler` pattern for webview handler then we would call `webviews.registerErrorHandler()` just below here. 

then we have a consistent pattern for both, and the setup is done at the same time (here) during startup.

Reply:
I've updated the code to match this pattern in a new commit.